Demo/Demo/ViewController.swift

```diff
@@ -3,8 +3,29 @@ import Card
 import PayPal
 
 class ViewController: UIViewController {
```

Comment:
The linter is complaining here for `Vertical Whitespace after Opening Braces Violation: Don't include vertical whitespace (empty line) after opening braces. (vertical_whitespace_opening_braces)`. I don't think we should follow this rule for class declarations, and a few other potential scenarios such as protocols, structs, etc.. Having a newline is common here & helps with organization.

Rather than make this PR conform to the linter, I think we can use this as a time to modify the linting rules for what we think is actually useful to lint on.

I'd rather remove this rule from the linter than need to add a `// swiftlint` comment on top of every class. Thoughts?

Reply:
+1 to removing that lint rule!!!

Reply:
No strong opinion from me - I prefer consistency over the linter, so maybe we can decide that we will always have an empty line at the top of protocols / classes ? I'm open to either, just want to do the same thing

Reply:
Removed this linter rule in PR #12. Agree with Jon on having this standardized to ensure it's consistent throughout the project!

Reply:
Awesome - thanks for taking care of that.